src/ripple/protocol/Indexes.h

```diff
@@ -283,6 +283,9 @@ xChainCreateAccountClaimID(STXChainBridge const& bridge, std::uint64_t seq);
 Keylet
 did(AccountID const& account) noexcept;
 
+Keylet
+oracle(AccountID const& account, std::uint32_t const& seq) noexcept;
```

Comment:
nit: I would call the param something other than `seq` - maybe something like `documentID` - so that it's clearer that it's not using the transaction sequence number like some of the other keylets do.